main.js

```diff
@@ -49,13 +47,14 @@ function returnMantra() {
     }
 };
 
-
 function returnMessage() {
     event.preventDefault();
     image.classList.add('hidden');
     messageText.classList.remove('hidden');
+    affirmationButton.disabled = true;
+    mantraButton.disabled = true;
     if (!affirmationButton.checked && !mantraButton.checked && !messageInput.value){
-        alert('Please select a message type!')
+        // alert('Please select a message type!')
```

Comment:
As this is a personal project, it doesn't matter as much, but out in the wild you will want to remove all commented out code before opening a PR that someone else will check.

Reply:
Thank you! That is incredibly helpful!
No commented code in PRs. Thank you!